dali/operators/audio/preemphasis_filter_op.cc

```diff
@@ -65,11 +65,11 @@ void PreemphasisFilterCPU::RunImplTyped(workspace_t<CPUBackend> &ws) {
   for (int sample_id = 0; sample_id < nsamples; sample_id++) {
     tp.AddWork(
       [this, &output, &input, sample_id](int thread_id) {
-        const auto in_ptr = input[sample_id].data<InputType>();
-        auto out_ptr = output[sample_id].mutable_data<OutputType>();
-        DALI_ENFORCE(input[sample_id].shape() == output[sample_id].shape(),
+        const auto in_ptr = input.tensor<InputType>(sample_id);
+        auto out_ptr = output.mutable_tensor<OutputType>(sample_id);
```

Comment:
Nitpick:
```suggestion
        auto *out_ptr = output.mutable_tensor<OutputType>(sample_id);
```
(for completeness)

Reply:
done